core/lib/NConnected.agda

```diff
@@ -98,10 +98,10 @@ conn-extend-general {B = B} {n = n} {k = S k'} {f = f} c P t = has-level-in
     Q : (g h : Π B (fst ∘ P)) → B → (k' +2+ n) -Type _
     Q g h b = ((g b == h b) , has-level-apply (snd (P b)) _ _)
 
-    app=-ap : ∀ {i j k} {A : Type i} {B : Type j} {C : B → Type k}
+    app=-∘ : ∀ {i j k} {A : Type i} {B : Type j} {C : B → Type k}
       (f : A → B) {g h : Π B C} (p : g == h)
       → app= (ap (λ k → k ∘ f) p) == (app= p ∘ f)
-    app=-ap f idp = idp
+    app=-∘ f idp = idp
```

Comment:
This name is good, and maybe `app=-post∘` or something like that could be better.

Reply:
I've renamed it to `app=-pre∘` since this lemma concerns the interaction between `app=` and *pre*composition with a function.

Reply:
Oops, you are right.